### PR TITLE
Blog post featuring everything released since Stately Studio 1.0

### PR DESCRIPTION
**Pull request by laurakalbag**
I’m still working on the images for this post, but I’d love to get feedback on:
- Whether I’ve covered everything we’ve released!
- If anything is incorrect or could be described better
- The order of the features. I’m thinking of moving the XState VSCode extension further up, any further suggestions much appreciated
- Anything else!

I’m also wondering how much of this can go into the newsletter; should I put it all in? Or shorten/summarise/cherry pick some key features?

### CODE REVIEW

**Comment by mellson** [comment]
@laurakalbag it's looking terrific! Some initial thoughts:

- There's a lot in here; should we split it across a series of blog posts?
- Your idea about moving the VS Code feature up is good. You could move it up top along with the Learn more with Stately streams.
- The new docs info could follow.
- Then, group the remaining under a header indicating that they all relate to the Stately Studio Editor.

File: content/posts/2023-01-20-whats-new-in-2023.mdx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  - Laura Kalbag
+ogImage: "https://stately.ai/blog/2023-01-20-whats-new-in-2023.png"
+publishedAt: "2023-01-20"
+---
+
+Happy 2023! The Stately team is back from their [first off-site of the year in Lisbon](https://twitter.com/statelyai/status/1615394794251096065) and excited to get started on our plans for this year. We thought we’d kick off the year with a reminder of all the features we’ve released since the [Stately Studio 1.0 release](https://stately.ai/blog/introducing-stately-studio-10) in October. It’s been just three months, and we already have so much to share.
+
+## Import from code
+
+[Import from code](https://stately.ai/blog/new-in-the-studio-import-from-code) was one of our most-requested features from users who had already created machines using XState or our Stately Viz. You can import an existing machine from JavaScript and TypeScript using the code panel in the editor’s right tool menu or the Machines list in the left drawer. Import from code works well with our existing export as code feature, where you can export your machine as JSON, JavaScript, and TypeScript using the code panel in the right tool menu.
```
Comment (farskid) [inline content/posts/2023-01-20-whats-new-in-2023.mdx:21]:
```suggestion
[Import from code](https://stately.ai/blog/new-in-the-studio-import-from-code) was one of our most-requested features from users who had already created machines using XState or our Stately Viz. You can import an existing machine from JavaScript, TypeScript or JSON using the code panel in the editor’s right tool menu or the Machines list in the left drawer. Import from code works well with our existing export as code feature, where you can export your machine as JSON, JavaScript, and TypeScript using the code panel in the right tool menu.
```

Comment (farskid) [reply content/posts/2023-01-20-whats-new-in-2023.mdx:21]: Plain JSON works now too :)

**Review by davidkpiano — APPROVED** [review]
🎉

**Pull request merged**